swift/test/Ice/objects/TestI.swift

```diff
@@ -77,59 +77,59 @@ class InitialI: Initial {
         _d.theC = nil  // Reference to a C.
     }
 
-    func getAll(current _: Ice.Current) async throws -> (b1: B?, b2: B?, theC: C?, theD: D?) {
+    func getAll(current _: Ice.Current) async throws -> sending (b1: B?, b2: B?, theC: C?, theD: D?) {
```

Comment:
I would remove the `async throws`. It's not clear to me how this works, since you're sending references to your properties.